SimplePolicy/SimplePolicy/config.json

```diff
@@ -2,6 +2,9 @@
   "PluginConfiguration": {
     "MaxTransactionsPerBlock": 500,
     "MaxFreeTransactionsPerBlock": 20,
+    "MaxTransactionSize": 100000,
+    "MaxFreeTransactionSize": 1024,
+    "FeePerExtraByte": 0.00001,
```

Comment:
Because it is not about priority, is about acceptable Tx.
In order to maintain free Tx we need to limit the usage of the blockchain to a responsible usage.
Most of the basic operations will still be free but if someone want to abuse of this will have to pay, proportionally.

Reply:
I think network fees are easier to reajust if necessary @jsolman, much easier than system fees... and the spirit I got is that everyone will have free tx, as long as they manage to keep below 1024 bytes. It's like 10 GAS fee, it incentives people to be "efficient" :) 
But I agree with you, on long-term conversations and more experiences, I believe we find another formula that is self-adaptive, but right now I think this is enough, in my opinion.